src/dss/initialize.py

```diff
@@ -0,0 +1,103 @@
+import os
+import time
+
+import yaml
+from charmed_kubeflow_chisme.kubernetes import KubernetesResourceHandler
+from lightkube import Client, KubeConfig
+from lightkube.resources.apps_v1 import Deployment
+from lightkube.resources.core_v1 import Namespace, PersistentVolumeClaim, Service
+
+from dss.logger import setup_logger
+
+# Set up logger
+logger = setup_logger("logs/dss.log")
+
+
+DSS_CLI_MANAGER_LABELS = {"app.kubernetes.io/managed-by": "dss-cli"}
+
+
+def wait_for_deployment_ready(
```

Comment:
Should we move this to a different file with common code? I would expect other future commands to need to use this (i.e. when creating a notebook/service)

Reply:
done